modules/ROOT/pages/app-security.adoc

```diff
@@ -27,6 +27,8 @@ ifdef::context[:parent-context: {context}]
 include::_partials/app-security/provisioning-security.adoc[leveloffset=+2]
 include::_partials/app-security/binding-an-app-to-a-service.adoc[leveloffset=+2]
 include::_partials/app-security/configuring-the-service.adoc[leveloffset=+2,tags=!excludeUpstream]
+include::_partials/app-security/setting-up-the-app-security-sdk.adoc[leveloffset=+2]
+include::_partials/app-security/proc-calling-the-app-security-clientInit.adoc[leveloffset=+2]
```

Comment:
```suggestion
== Using the {app-security-service} Service
include::_partials/app-security/proc-calling-the-app-security-clientInit.adoc[leveloffset=+2]
```

Does it make more sense to put this under its own header? ^

Reply:
not sure I understand, isn't that heading already in the partial?

Reply:
agree it would be a duplicate heading , maybe set the [leveloffset=+1] which would make it more prominent

Reply:
had a look and it doesn't look right 
![image](https://user-images.githubusercontent.com/16667688/55385060-32bde480-5524-11e9-840f-c558dd702d94.png)

will leave it as is